### PR TITLE
Default url rules

### PR DESCRIPTION
**Pull request by artem-fomin-by**
Allowed to specify default browsers for urls and url patterns

### CODE REVIEW

File: MainProgramTests/WildcardTests.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+namespace MainProgramTests;
+
+internal class WildcardTests
+{
+    private static bool HasPreviousTestPassed = true;
+
+    [SetUp]
+    public void SetUp()
+    {
+        Assume.That(HasPreviousTestPassed, Is.True);
```
Comment (alex-fomin) [inline MainProgramTests/WildcardTests.cs:14]: Why do we need this?

File: MainProgramTests/WildcardTests.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    public void ConvertingTest()
+    {
+        var regexPattern = @".*\.youtube\.com/.*";
+        var wildcard = @"*.youtube.com/*";
+        var wildcardRegex = WildcardUtilities.WildcardToRegex(wildcard);
+        Assert.AreEqual(regexPattern, wildcardRegex.ToString());
+    }
+
+    [Test, Order(2)]
+    public void MatchTest()
```
Comment (alex-fomin) [inline MainProgramTests/WildcardTests.cs:34]: It is better to use TestCase here: https://docs.nunit.org/articles/nunit/writing-tests/attributes/testcase.html

And Orders are not required

File: MainProgramTests/WildcardTests.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    {
+        var browser = new Browser{ UrlPatterns = new[]{ @"*.youtube.com/*" } };
```
Comment (alex-fomin) [inline MainProgramTests/WildcardTests.cs:36]: Also please add more cases - without *, without * on the end, etc.

**Review by alex-fomin — CHANGES_REQUESTED** [review]

File: MainProgramTests/WildcardTests.cs (hunk 1 above)
Comment (artem-fomin-by) [reply MainProgramTests/WildcardTests.cs]: Resolved in new commit

File: MainProgramTests/WildcardTests.cs (hunk 2 above)
Comment (artem-fomin-by) [reply MainProgramTests/WildcardTests.cs]: Resolved in new commit

File: MainProgramTests/WildcardTests.cs (hunk 3 above)
Comment (artem-fomin-by) [reply MainProgramTests/WildcardTests.cs]: Resolved in new commit

**Review by alex-fomin — APPROVED** [review]

**Pull request merged**